manifests/plugin/oracle/database.pp

```diff
@@ -17,7 +17,7 @@
   validate_array($query)
 
   concat::fragment { "collectd_plugin_oracle_database_${name}":
-    order   => '10',
+    order   => '20',
```

Comment:
Collectd sends errors to syslog on every collect. I havn't the full messages here.